oeps/oep-0005-arch-containerize-devstack.rst

```diff
@@ -232,6 +232,21 @@ environment, and was the prototype implementation of this OEP. However, it
 will need modification to fully meet the specifications of this OEP.
 
 
+Decisions
+=========
+
+Multiple decisions have been made that either modify or append to this OEP.
+The hope is for these decisions to eventually make their way back to this OEP,
+but while they are not, links to decisions are listed below:
```

Comment:
* Do these decisions really modify the OEP?  Would it be just as simple to add explicit links to the decisions in context in the OEP (in addition or in place of this list)?
* What about the following alternative sentence (note the intentional lack of line breaks):
```
The following related decisions modify or enhance this OEP, but have not yet been fully incorporated as updates to this OEP:
```

Reply:
I like your modification to OEP and will add it in next commit.

As for putting explicit links in context of OEP, I opted not to do that, cause:
1. its a-lot of work
2. This whole OEP needs to be updated and these decisions don't really do it well.
3. This changes moves us forward in terms of visibility, so I opted to just do this quickly and hopefully someone else will come back and make the whole OEP better.